server/pulp/server/webservices/urls.py

```diff
@@ -88,8 +87,11 @@
     url(r'^v2/roles/(?P<role_id>[^/]+)/$', RoleResourceView.as_view(), name='role_resource'),
     url(r'^v2/roles/(?P<role_id>[^/]+)/users/$', RoleUsersView.as_view(), name='role_users'),
     url(r'^v2/roles/(?P<role_id>[^/]+)/users/(?P<login>[^/]+)/$', RoleUserView.as_view(), name='role_user'),
-    url(r'^v2/tasks/$', TaskCollectionView.as_view(), name='task_collection'),
-    url(r'^v2/tasks/(?P<task_id>[^/]+)/$', TaskResourceView.as_view(), name='task_resource'),
-    url(r'^v2/users/$', UsersView.as_view(), name='users'),
-    url(r'^v2/users/(?P<login>[^/]+)/$', UserResourceView.as_view(), name='user_resource')
+    url(r'^v2/tasks/$', tasks.TaskCollectionView.as_view(), name='task_collection'),
```

Comment:
On 03/19/2015 07:51 AM, Ina Panova wrote:
> is there a reason why you changed from TaskCollectionView.as_view() to
> tasks.TaskCollectionView.as_view() ?
> Is not it more readable the first option?

I personally prefer having my imports be against modules, so that when I
use items from other modules everything is name spaced. There are some
times this becomes much cleaner. For example, we have a
tasks.serialize_task(), which is redundant since it's in tasks.py. For
users, I created a users.serialize(), which I think is much nicer.

It is a matter of preference of course, so I wouldn't argue one way or
the other on other people's code. I think it's cleaner to look at code
that gives me a clue about which module function calls and classes came
from.

-- 
Randy Barlow